XamlControlsGallery/ControlPages/ButtonPage.xaml

```diff
@@ -18,13 +18,12 @@
     <StackPanel>
         <local:ControlExample x:Name="Example1" HeaderText="A simple Button with text content.">
             <local:ControlExample.Example>
-                <RelativePanel>
-                    <Button x:Name="Button1" Content="Standard XAML button" Click="Button_Click"
-                            IsEnabled="{x:Bind DisableButton1.IsChecked.Value.Equals(x:False), Mode=OneWay}" />
-                    <TextBlock x:Name="Control1Output" Margin="8,0,0,0" FontFamily="Global User Interface"
-                            RelativePanel.RightOf="Button1" RelativePanel.AlignVerticalCenterWith="Button1" />
-                </RelativePanel>
+                <Button x:Name="Button1" Content="Standard XAML button" Click="Button_Click"
+                    IsEnabled="{x:Bind DisableButton1.IsChecked.Value.Equals(x:False), Mode=OneWay}" />
             </local:ControlExample.Example>
+            <local:ControlExample.Output>
+                <TextBlock x:Name="Control1Output" FontFamily="Global User Interface"/>
+            </local:ControlExample.Output>
```

Comment:
I am not sure if this "styling" is ideal. Maybe we can move the output somewhere else? The top sample looks a bit strange in my opinion.

![image](https://user-images.githubusercontent.com/16122379/88849142-bf94b300-d1e9-11ea-942e-a6c2f83f3be9.png)

Reply:
Agreed that the styling is not ideal - but not sure how to fix it. Open to suggestions! We'll go with this styling for now, but we can always update it later.